src/main/host/network_queuing_disciplines.c

```diff
@@ -25,7 +25,7 @@ void rrsocketqueue_destroy(RrSocketQueue* self, void (*fn_processItem)(const Com
 
     if (fn_processItem != NULL) {
```

Comment:
It looks like this is a new file, or did I miss it getting moved from somewhere else? Does this belong in this PR? It looks like it's untested...?

Reply:
This is new code, and any of the round-robin qdisc methods aren't used in our tests. I've verified the round-robin qdisc manually by running phold with `--interface-qdisc=RR` and the tests pass. I've also now added another phold configuration test using this option.

The only thing weird to me in the coverage results is that `_compareSocketTagged()` isn't being run, which means there might be an issue with the priority queue. But I haven't made any changes to the priority queue, so I don't know what's going on there.